templates/kubernetes/terraform/environments/stage/main.tf

```diff
@@ -80,4 +80,6 @@ module "kubernetes" {
   frontend_service_domain     = "<% index .Params `stagingFrontendSubdomain` %>${local.domain_name}"
   # This domain or address must be verified by the mail provider (Sendgrid, SES, etc.)
   user_auth_mail_from_address = "noreply@${local.domain_name}"
+
+  use_notification_service    = <%if eq (index .Params `sendgridApiKey`) "" %>false<% else %>true<% end %>
```

Comment:
maybe `notification_service_enabled` ? 

we have a bunch of flags like 
`cf_signing_enabled ` (also declared in local as `file_uploads`)  
`sendgrid_enabled`
`auth_enabled`